docs/configuration/config-file-reference.md

```diff
@@ -1899,6 +1899,51 @@ bucket_store:
   # CLI flag: -blocks-storage.bucket-store.series-batch-size
   [series_batch_size: <int> | default = 10000]
 
+  token_bucket_limiter:
+    # Whether token bucket limiter is enabled
+    # CLI flag: -blocks-storage.bucket-store.token-bucket-limiter.enabled
+    [enabled: <boolean> | default = false]
+
+    # Whether the token bucket limiter is in dry run mode
+    # CLI flag: -blocks-storage.bucket-store.token-bucket-limiter.dry-run
+    [dry_run: <boolean> | default = false]
+
+    # Instance token bucket size
+    # CLI flag: -blocks-storage.bucket-store.token-bucket-limiter.instance-token-bucket-size
+    [instance_token_bucket_size: <int> | default = 859832320]
+
+    # User token bucket size
+    # CLI flag: -blocks-storage.bucket-store.token-bucket-limiter.user-token-bucket-size
+    [user_token_bucket_size: <int> | default = 644874240]
+
+    # Request token bucket size
+    # CLI flag: -blocks-storage.bucket-store.token-bucket-limiter.request-token-bucket-size
+    [request_token_bucket_size: <int> | default = 4194304]
+
+    # Multiplication factor used for fetched postings token
```

Comment:
Is the token calculated using fetched postings bytes or count? I think it is multiplied by bytes but from the doc it looks like by count.
Can we mention `bytes` somewhere or just call it `token-bucket-bytes-limiter`?

Reply:
will do!